### ISSUE TITLE
Add support for FoundryVTT CLI

### ISSUE THREAD

**Issue by felddy** [felddy/foundryvtt-docker#654]
### Feature description

Include the FoundryVTT CLI in the image.  Document the usage of the command from the host system. 

See:
- https://github.com/foundryvtt/foundryvtt-cli

### Motivation

- This would support the development community that uses the container.
- Removes the need to developers to modify the container.
- Makes the container more useful in a development workflow.

### Example

TBD

### Pitch

It improves the usefulness of the container without causing bloat or complication to existing use cases.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by orangepeelbeef** [comment]
this really should be 3 things to get fvtt cli working in container

1. npm install -g @foundryvtt/foundryvtt-cli
2. fvtt configure set installPath /the/installPath
3. fvtt configure set dataPath /the/dataPath

**Comment by felddy** [comment]
Agreed that this should be relatively straightforward to implement.

I have a concern about how the tool stores its configuration in the `root` user's home directory.  This could be problematic for people that are not running the container as `root`.  

https://github.com/foundryvtt/foundryvtt-cli/blob/8c975536fb9e748274f6910790f43581d9740071/config.mjs#L33-L41

```console
# cat ~root/.local/share/.fvttrc.yml 
installPath: /home/foundry/resources/app
dataPath: /data
```

There is a feature request open that, if implemented, will add some needed flexibility to the configuration location:
- https://github.com/foundryvtt/foundryvtt-cli/issues/40

**Comment by orangepeelbeef** [comment]
Hm, I didn't notice that, but I see it now on my system as well.  

Looking at that code it seems like there are 2 options
set XDG_DATA_HOME env variable to where you want the config to go

or
execute fvtt as the same userid you are using for the container (which drops the file as /home/foundry/.fvttrc.yml)

/home/foundry $ ls -alh
total 172K   
drwxr-sr-x    1 foundry  foundry      111 Oct 25 06:56 .
drwxr-xr-x    1 root     root          33 Sep 29 10:07 ..
-rw-r--r--    1 root     foundry        3 Oct 25 06:48 .fvttrc.yml

**Comment by insensitiveclod** [comment]
I was looking into using foundryvtt-cli as a good way to auto pack/unpack data from a module and I thought it'd be nice to have it just be included with the docker-image. 
Found this issue; great!

The feature request for local config (via a `--config` option) has been integrated into `foundryvtt-cli` in https://github.com/foundryvtt/foundryvtt-cli/pull/50

Could you pull it in ? I can help test.
Cheers!

**Comment by BfdZen6** [comment]
Now that npm is removed from the image, I can't find a simple way to install foundryvtt-cli to a felddy/foundryvtt container anymore (I used to install it it after container creation as a simple workaround).
Could you please add foundryvtt-cli to the image now that the expected conditions seems to be met ?
Thanks a lots for you work !

**Comment by felddy** [comment]
The development build of the image contains the CLI: `ghcr.io/felddy/foundryvtt:develop`

The docs are in the readme:
- https://github.com/felddy/foundryvtt-docker#foundry-vtt-cli

Let me know if this solution works for y'all.
